entrypoint.sh

```diff
@@ -15,5 +15,13 @@ if [ "$1" == "install" ]; then
     exit
   fi
 else
-  exec kube-bench "$@"
+  if [ -n "$SCHEDULE" ]; then
```

Comment:
would it make more sense to make schedule a flag rather than an environment variable? just from a UX perspective, this is an entirely different mode of operation (of the entrypoint)

Reply:
A CLI flag sounds like a good way. I could make this change if that's preferred.